apps/web/src/components/accessibility/accessibility-enhanced-components.tsx

```
@@ -0,0 +1,566 @@
+                  isMedical && 'text-blue-800'
+                )}
+              >
+                {title}
+              </h2>
+            )}
+            <button
+              onClick={onClose}
+              className="text-gray-500 hover:text-gray-700 text-2xl leading-none"
+              aria-label="Close modal"
+            >
+              ×
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)